src/mds/MDSTable.h

```diff
@@ -27,7 +27,7 @@ class MDSTable {
 public:
   MDSRank *mds;
 protected:
-  const char *table_name;
+  std::string table_name;
```

Comment:
why not just use string_view? as we assume the content of string_view is always null terminated.

Reply:
`MDSTable` is only statically instantiated a few times in the MDS so the cost here is low.

I'm also concerned about having a `const char *` or `std::string_view` which refers to memory the class does not control/reference.